### PR TITLE
addSeries test and website changes

### PR DESCRIPTION
**Pull request by illetid**
Fix tests and docs series code
**Type of PR:** enhancement

**PR checklist:**

- [ ] Addresses an existing issue: fixes #
- [x] Includes tests
- [x] Documentation update

### CODE REVIEW

**Comment by SlicedSilver** [comment]
Can we add the new series exports as defined entries within: `website/tutorials/how_to/.eslintrc.js` so that we don't need to use `// eslint-disable-next-line no-undef` in files like `website/tutorials/how_to/two-price-scales.js`

**Comment by illetid** [comment]
> Can we add the new series exports as defined entries within: `website/tutorials/how_to/.eslintrc.js` so that we don't need to use `// eslint-disable-next-line no-undef` in files like `website/tutorials/how_to/two-price-scales.js`

added series to globals, thanks

File: website/docs/migrations/from-v4-to-v5.md [hunk 1]
```diff
@@ -3,6 +3,26 @@
...
+const chart = createChart(container, {});
+const lineSeries = chart.addSeries(LineSeries, { color: 'red' });
+```
+
+UMD:
+
+```js
+const chart = LightweightCharts.createChart(container, {});
+const lineSeries = chart.addSeries(LightweightCharts.LineSeries, { color: 'red' });
+```
```
Comment (SlicedSilver) [inline website/docs/migrations/from-v4-to-v5.md:24]: Could we make this a bit more explicit? I feel that using the term SeriesDefinition might confuse some people (and that they would expect there to be an export named `SeriesDefinition`).

I've attached a md file with a suggestion.
[series-changes-section.md](https://github.com/user-attachments/files/17704849/series-changes-section.md)

Comment (illetid) [reply website/docs/migrations/from-v4-to-v5.md]: changed, thanks for the suggestion

**Review by SlicedSilver — APPROVED** [review]

**Pull request merged**